src/object.d

```diff
@@ -1571,6 +1603,10 @@ const:
         return null;
     }
 
+    /****************
+     * Returns:
+     *  array of TypeInfo_Class references for classes defined in this module
```

Comment:
"TypeInfo_Class" should probably be a link.  For example `$(DDLINK	phobos/object, TypeInfo_Class,
$(D object.TypeInfo_Class))`